extensions/compose/src/extension.ts

```diff
@@ -468,6 +468,36 @@ async function deleteFileAsAdmin(filePath: string): Promise<void> {
   }
 }
 
+async function deleteExecutableAsAdmin(filePath: string): Promise<void> {
+  const system = process.platform;
```

Comment:
issue: process.platform should not be used inside an extension, there is env.isWindows, env.isMac, etc through the exposed API